index.d.ts

```diff
@@ -0,0 +1,23 @@
+declare module 'react-native-toast-message' {
+  import React from 'react';
+  const Toast: {
+    show(options: {
+      type?: 'success' | 'error' | 'info';
+      position?: 'top' | 'bottom';
+      text1: string;
+      text2: string;
+      visibilityTime?: number;
+      autoHide?: boolean;
+      topOffset?: number;
+      bottomOffset?: number;
+      props: { onPress: () => any; guid: string };
```

Comment:
same for `props`, optional, with an object literal shape
```ts
interface ObjectLiteral {
  [key: string]: any;
}
```
```suggestion
      props?: ObjectLiteral;
```